src/kebab.rs

```diff
@@ -17,9 +18,19 @@ pub trait ToKebabCase: ToOwned {
     fn to_kebab_case(&self) -> Self::Owned;
 }
 
+pub fn to_kebab(s: &str, number_starts_word: bool) -> String {
```

Comment:
With the proposed changes to the `convert_case` module, this function doesn't need to exist.

Reply:
Same for the other modules.